winum.el

```diff
@@ -447,7 +447,7 @@ windows, however a higher number can be reserved by the user-defined
        (vconcat window-vector
                 (make-vector (1+ (- number window-vector-length)) nil))))))
 
-(defun winum--window-list ()
+(defun winum--get-window-list ()
```

Comment:
I was that `winum--window-list` was dealing with the scope so I put the check in, that's why I renamed it - to be consistent with the other getter functions that need checking.

You're right that the check is not needed here, so the name needn't be changed either.